tests/components/litejet/test_trigger.py

```diff
@@ -73,7 +73,7 @@ async def simulate_press(hass, mock_lj, number):
         return_value=mock_lj.start_time + mock_lj.last_delta,
     ):
         if callback is not None:
-            await hass.async_add_job(callback)
+            await hass.async_add_executor_job(callback)
```

Comment:
This is correct at the moment. But the callbacks could be rewritten to run in async context. There's no need to have them run in sync mode:
https://github.com/home-assistant/core/blob/5e96d2141410680f369202065b3c7fd526e0e7df/homeassistant/components/litejet/trigger.py#L39-L95

Reply:
Lol, scratch that, they are called from sync context. It's Friday.